app/app.py

```diff
@@ -42,8 +43,9 @@ def run_kubeseal():
         cltSecret = form.cleartextSecret.data
         sName = form.secretName.data
         sNamespace = form.secretNamespace.data
+        sencryptedDataKeyName = form.encryptedDataKeyName.data
         sealedSecret = Kubeseal.kubectlCMD(cltSecret, sNamespace, sName)
-        log.info('Created SealedSecret [%s] for Namespace[%s]', sName, sNamespace)
+        log.info('Created SealedSecret [%s] for Namespace[%s] with EncryptedData-Key-Name[%s]', sName, sNamespace,sencryptedDataKeyName)
```

Comment:
- Im Englischen neigt man eher dazu Wörter ohne Bindestrich zu schreiben, so z.B. "EncryptedData key name". 
- Ich persönlich würde dann "encrypted data" auch klein und getrennt schreiben.
- Substantive im Satz werden auch klein geschrieben, wenn es nicht gerade Eigennamen sind. Also Namespace würde ich tendenziell auch klein schreiben. SealedSecret als spezieller Name des Objekts kann man durchaus so lassen.
- Leerzeichen sind ein bisschen durcheinander.

Am Ende dann m.E. so:
'Created SealedSecret [%s] for namespace [%s] with encrypted data key name: [%s].'